insights/specs/default.py

```diff
@@ -473,6 +477,9 @@ class DefaultSpecs(Specs):
     password_auth = simple_file("/etc/pam.d/password-auth")
     pci_rport_target_disk_paths = simple_command("/usr/bin/find /sys/devices/ -maxdepth 10 -mindepth 9 -name stat -type f")
     pcp_metrics = simple_command("/usr/bin/curl -s http://127.0.0.1:44322/metrics --connect-timeout 5", deps=[pcp_enabled])
+    # TODO: "/var/log/pcp/pmlogger/pcpdata.tar.xz" needs to be confirmed
+    pcp_raw_data = simple_file("/var/log/pcp/pmlogger/pcpdata.tar.xz",
```

Comment:
This file won't be readily available, we would need to create it out of YYYYMMDD.index, YYYYMMDD.meta.xz, YYYYMMDD.N.xz files, if this is what you are waiting on.

I am also curious if this should be simple_file or temp file, because this is not required on the system once we include it in insights-archive.

Reply:
Hi @upadhyeammit - Thanks for reviewing and sorry for reply to you late.

`simple_file` is just a helper function for insights core collection, it's good enough to collect this file into the insights-archive. TBH, it's not so exact as in insights the `simple_file` always indicates a `text` file 😓 

> because this is not required on the system once we include it in insights-archive.
Do you mean, that you want the insights core to remove the file after the collection?